lib/configParser.js

```diff
@@ -96,12 +96,18 @@ ConfigParser.resolveFilePatterns =
 
   if (patterns) {
     for (var i = 0; i < patterns.length; ++i) {
-      var matches = glob.sync(patterns[i], {cwd: cwd});
+      var fileName = patterns[i].split(':')[0];
+      var lineNumber = patterns[i].split(':')[1];
+      var matches = glob.sync(fileName, {cwd: cwd});
       if (!matches.length && !opt_omitWarnings) {
         log.warn('pattern ' + patterns[i] + ' did not match any files.');
       }
       for (var j = 0; j < matches.length; ++j) {
-        resolvedFiles.push(path.resolve(cwd, matches[j]));
+        if(lineNumber) {
```

Comment:
nit: clear the redundancy a little, and it makes it more obvious that lineNumber is only for cucumber. 

var resolvedPath = path.resolve(cwd, matches[j]);
if (lineNumber) {
  resolvedPath += ':' + lineNumber;
}
resolvedFiles.push(resolvedPath);